config/runtime/mux.go

```diff
@@ -3,27 +3,27 @@ package runtime
 import (
 	"net/http"
 
+	"github.com/avenga/couper/config"
 	"github.com/avenga/couper/errors"
 )
 
 type MuxOptions struct {
 	EndpointRoutes map[string]http.Handler
 	FileRoutes     map[string]http.Handler
 	SPARoutes      map[string]http.Handler
-	ErrorTpl       *errors.Template
-	Hosts          hosts
+	APIErrorTpls   map[*config.API]*errors.Template
```

Comment:
Use the server options struct as field option instead of remapping all related fields.

Reply:
Done